### PR TITLE
[OSDEV-693] Add the 'test' environment to ecsmanage configuration

### PR DESCRIPTION
**Pull request by Innavin369**
Add the 'test' environment to ecsmanage configuration

### CODE REVIEW

**Review by killev — CHANGES_REQUESTED** [review]
Please add information to RELEASE-NOTES

File: src/django/oar/settings.py [hunk 1]
```diff
@@ -24,55 +23,55 @@
...
-    ALLOWED_HOSTS.append('app')
-    ALLOWED_HOSTS.append('contricleaner')
+if ENVIRONMENT == "Development":
+    ALLOWED_HOSTS.append("localhost")
+    ALLOWED_HOSTS.append("django")
+    ALLOWED_HOSTS.append("app")
+    ALLOWED_HOSTS.append("contricleaner")
 
-if ENVIRONMENT in ['Production', 'Staging', "Prestaging"] and BATCH_MODE == '':
+if ENVIRONMENT in ["Production", "Staging", "Prestaging"] and BATCH_MODE == "":
```
Comment (vladsha-dev) [inline src/django/oar/settings.py:74]: Let's delete Prestaging.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
1. I left one comment.
2. There are two more files where changes need to be made. See the screenshot.

Screenshot 2024-03-04 at 17 06 52

**Review by killev — APPROVED** [review]
LGTM. Approved

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**